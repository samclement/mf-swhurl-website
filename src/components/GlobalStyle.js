import { createGlobalStyle } from 'styled-components'

const GlobalStyle = createGlobalStyle`
  body { font-family: Helvetica, Arial, sans-serif; }
  .touch { -webkit-overflow-scrolling: touch; }
  .no-scroll { position: fixed; }
`

export default GlobalStyle
