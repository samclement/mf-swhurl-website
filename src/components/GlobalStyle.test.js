/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'
import 'jest-styled-components'
import '@testing-library/jest-dom/extend-expect'
import GlobalStyle from './GlobalStyle.js'

describe('With Snapshot Testing', () => {
  it('Global Style', () => {
    render(<GlobalStyle />)
    const styleTags = document.head.getElementsByTagName('style')
    expect(styleTags).toMatchSnapshot()
  })
})
