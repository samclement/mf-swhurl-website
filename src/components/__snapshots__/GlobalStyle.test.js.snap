// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Global Style 1`] = `
HTMLCollection [
  <style
    data-styled="active"
    data-styled-version="5.0.1"
  >
    body{font-family:Helvetica,Arial,sans-serif;}
    .touch{-webkit-overflow-scrolling:touch;}
    .no-scroll{position:fixed;}
  </style>,
]
`;
